generated/nidaqmx/_task_modules/in_stream.py

```diff
@@ -330,22 +331,23 @@ def input_onbrd_buf_size(self):
         return val
 
     @property
-    def logging_file_path(self):
+    def logging_file_path(self) -> pathlib.PurePath:
```

Comment:
My suggestion was to return `pathlib.Path` here: 
> InStream.logging_file_path getter returns pathlib.Path

The difference is that `pathlib.Path` has methods that support filesystem access and `pathlib.PurePath` omits these methods.

The most versatile approach is to accept `PurePath`, but return `Path`.

Reply:
My mistake, mistook your suggestion `pathlib.Path` to `pathlib.PurePath`. The return type is corrected as per suggestion.